README.md

```diff
@@ -92,15 +92,15 @@ method     | url                                                    | file path
 
 ### Nested Resources
 
-Nesting of one layer at most is supported.
+Nesting of two layer at most is supported.
 
 method     | url                                                                       | file path                         | controller name
 ---        | ---                                                                       | ---                               | ---
-**GET**    | `/doc/api/{parents}/:parent_id/{objects}?per_page={per_page}&page={page}` | `app/apis/{parents}/{objects}.js` | **index()**
-**GET**    | `/doc/api/{parents}/:parent_id/{objects}/:id`                             | `app/apis/{parents}/{objects}.js` | **show()**
-**POST**   | `/doc/api/{parents}/:parent_id/{objects}`                                 | `app/apis/{parents}/{objects}.js` | **create()**
-**PUT**    | `/doc/api/{parents}/:parent_id/{objects}/:id`                             | `app/apis/{parents}/{objects}.js` | **update()**
-**DELETE** | `/doc/api/{parents}/:parent_id/{objects}/:id[s]`                          | `app/apis/{parents}/{objects}.js` | **destroy()**
+**GET**    | `/doc/api/{parents}/:parent_id/{children}/:child_id/{objects}?per_page={per_page}&page={page}` | `app/apis/{parents}/{objects}.js` | **index()**
```

Comment:
@popomore 

It is indeed in this format `apis/{parents}/{children}/{objects}.js`

I have updated the readme two days ago based on @fengmk2 's comment. 

[See here](https://github.com/eggjs/egg-rest/blob/feature/nested/README.md#nested-resources)